Python/pywarpx/picmi.py

```diff
@@ -2404,6 +2404,7 @@ class ParticleDiagnostic(picmistandard.PICMI_ParticleDiagnostic, WarpXDiagnostic
     def init(self, kw):
 
         self.format = kw.pop('warpx_format', 'plotfile')
+        self.type = kw.pop('warpx_diag_type', 'Full')
```

Comment:
Awesome, great work! 

Can you please just a few lines above, in the documentation string, add an entry for the new `warpx_diag_type` option? That way it will be shown in our manual automatically and users will find it.
https://warpx.readthedocs.io/en/latest/usage/python.html#diagnostics